net/aria2/Makefile

```diff
@@ -15,21 +15,27 @@ PKG_SOURCE_URL:=https://github.com/aria2/aria2/releases/download/release-$(PKG_V
 PKG_HASH:=546e9194a9135d665fce572cb93c88f30fb5601d113bfa19951107ced682dc50
 PKG_INSTALL:=1
 
-PKG_MAINTAINER:=Imre Kaloz <kaloz@openwrt.org>, Hsing-Wang Liao <kuoruan@gmail.com>
+PKG_MAINTAINER:=Imre Kaloz <kaloz@openwrt.org>, \
+	Hsing-Wang Liao <kuoruan@gmail.com>
 PKG_LICENSE:=GPLv2
 PKG_LICENSE_FILES:=COPYING
 
 PKG_CONFIG_DEPENDS := \
-  ARIA2_NOSSL \
-  ARIA2_OPENSSL \
-  ARIA2_GNUTLS \
-  ARIA2_BITTORRENT \
-  ARIA2_SFTP \
-  ARIA2_ASYNC_DNS \
-  ARIA2_COOKIE \
-  ARIA2_METALINK \
-  ARIA2_EXPAT \
-  ARIA2_LIBXML2
+	ARIA2_NOSSL \
+	ARIA2_OPENSSL \
+	ARIA2_GNUTLS \
+	ARIA2_NOCRYPTO \
+	ARIA2_NETTLE \
+	ARIA2_LIBGCRYPT \
+	ARIA2_LIBXML2 \
+	ARIA2_EXPAT \
+	ARIA2_GMP \
+	ARIA2_BITTORRENT \
+	ARIA2_METALINK \
+	ARIA2_SFTP \
+	ARIA2_ASYNC_DNS \
+	ARIA2_COOKIE \
+	ARIA2_WEBSOCKET
```

Comment:
Hmm, I cannot find the code doing it in aria2/Makefile, include/package.mk, rules.mk .  Can you shed more light ? ;)